examples/knowledge/cmd/property.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2023 IndyKite
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	knowledgeobjects "github.com/indykite/indykite-sdk-go/gen/indykite/knowledge/objects/v1beta1"
+	knowledgepb "github.com/indykite/indykite-sdk-go/gen/indykite/knowledge/v1beta2"
+	objects "github.com/indykite/indykite-sdk-go/gen/indykite/objects/v1beta1"
+)
+
+// propertyCmd represents the command for making a listNodesByProperty query with the Identity Knowledge API
+var propertyCmd = &cobra.Command{
+	Use:   "property",
+	Short: "Make a list query to the IndyKite Identity Knowledge API",
+	Long:  `Make a list query to the IndyKite Identity Knowledge API`,
+	Args:  cobra.ExactArgs(0),
+	Run: func(cmd *cobra.Command, args []string) {
+
+		query := "MATCH (n:Resource)-[:HAS]->(p:Property) WHERE p.type='last_name' and p.value='mushu'"
```

Comment:
rather use the input params properly instead `p.type = $type and p.value = $value` and then provide `type` and `value` in the input params

Reply:
updated